huaweicloud/services/apig/resource_huaweicloud_apig_api.go

```diff
@@ -884,6 +918,13 @@ func buildApiType(t string) int {
 	}
 }
 
+func isObjectEnabled(isEnabled bool) int {
```

Comment:
The function name `isXXXEnabled` is more like returen boolean value.
How about rename to `convertBoolToInt()`

Reply:
not good.
For the API design of the APIG service, they always use the number `1` to explain yes and use numebr `2` to explain no, normally, the yes is `1` and the no is `0`, but the APIG API is not like this.
So, I think `convertBoolToInt()` is not good.